python_modules/libraries/dagster-gcp/dagster_gcp/bigquery/io_manager.py

```diff
@@ -249,6 +249,7 @@ def type_handlers() -> Sequence[DbTypeHandler]:
     def default_load_type() -> Optional[Type]:
         return None
 
+    # NOTE to reviewers. This is where having the factory comes in hand to do the with block.
```

Comment:
Here is the one that was difficult/impossible to convert while maintain existing semantics.